scss/_base_forms.scss

```
@@ -106,6 +106,10 @@
       position: relative;
     }
 
+    &.u-truncate {
+      width: auto;
```

It might not be obvious why we're doing this at first glance, so we should add a comment explaining why.
```suggestion
      // width: fit-content prevents ellipses when u-truncate is applied
      width: auto;
```